analysis/wwz/make_datacards.py

```diff
@@ -251,7 +283,8 @@ def get_syst_base_name_lst(in_lst):
                 valvar_kappa_do = yt.valvar_op(valvar_do,valvar_nom,"div")
 
                 # Handle negative cases
-                if (valvar_kappa_up[0]<=0) and (valvar_kappa_do[0]<=0): raise Exception("Both kappas negative, should not be possible.")
+                if (valvar_kappa_up[0]<=0) and (valvar_kappa_do[0]<=0):
```

Comment:
This seems to only check the negative direction?

Reply:
Yes, but this is what we had before. Would we want to change this?

Reply:
Let's revisit later. See https://github.com/cmstas/ewkcoffea/issues/45